test/evergreen.yml

```diff
@@ -1952,6 +1973,20 @@ tasks:
           format_test_script_args: -b "SEGFAULT_SIGNALS=all catchsegv ./t" -S
           times: 16
 
+  - name: checkpoint-stress-test
+    commands:
+      - command: timeout.update
+        params:
+          # set 24 hour timeout for the task and command
+          exec_timeout_secs: 86400
+          timeout_secs: 86400
```

Comment:
Thanks for figuring out the way to customize both timeout settings. 
It would be nice to remind Evergreen team to add this command into the wiki page.

Reply:
> Thanks for figuring out the way to customize both timeout settings.
> It would be nice to remind Evergreen team to add this command into the wiki page.

Yeah, sure. I will do that.